app/decorators/homepage.rb

```diff
@@ -25,7 +25,7 @@ def admin_posts
       @admin_posts = AdminPost.non_translated.for_homepage.all
     else
       @admin_posts = Rails.cache.fetch("home/index/home_admin_posts", expires_in: 20.minutes) do
-        AdminPost.non_translated.for_homepage.all
+        AdminPost.non_translated.for_homepage
```

Comment:
Using `all` results in a deprecation warning. Without all, it returns  an ActiveRecord::Relation object. With all it is returning an array. If the array is the desired output, we can use `to_a` to get the same result without a deprecation warning. Does that work?